pycheribuild/config/chericonfig.py

```diff
@@ -222,6 +222,20 @@ def __init__(self, loader: ConfigLoaderBase, action_class):
         self.wait_for_debugger = loader.add_bool_option("wait-for-debugger", group=loader.run_group,
                                                         help="Start QEMU in the 'wait for a debugger' state when"
                                                              "launching CheriBSD,FreeBSD, etc.")
+
+        self.debug_kernel = loader.add_bool_option("debug-kernel",
```

Comment:
I'd prefer a different name for this option. Maybe `debugger-in-new-tmux-pane` but there might be a shorter and better name.